src/components/ToDoListItem.js

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import {Checkbox} from "@cdk-uip/react-checkbox";
+import { TextField } from "@cdk-uip/react-text-field";
+
+import { TextFieldIcon } from "@cdk-uip/react-text-field"
+
+
+class ToDoListItem extends React.Component{
+   constructor(props){
+       super(props)
+       console.log("");
+   }
+
+    render() {
+        
+
+        return (
+            <span>
+             
+                    <Checkbox
+                        checked={this.props.item.checked}
+                        value={this.props.item.value}
+                        onChange={e => {
+                            
+                            this.props.updateCheck(this.props.index, e.target.checked, e.target.value);
+
+                        }
+                    
+                    }
+                    />
+                
+              
+                <TextField
+                    label="Your Task"
+                    value={this.props.item.value}
+                    disabled = { this.props.item.disabled}
+                    onChange={e => {
+                            this.props.updateV(this.props.index, e.target.value);
+                        }}
+                   
+                />
+                <TextFieldIcon 
+                    onClick={e => {
```

Comment:
Generally only calling one thing like a function no need for extra `{}'s`

Just do: `onChange={e => this.props.delete(this.props.index)}`